### DISCUSSION TITLE
Help with Home Assisstant

### DISCUSSION THREAD

**Discussion by ronnaudao** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#695] [Q&A]
Hello. I installed this mod in a miami 2021 with d1 mini. Its everything working just fine, but im having some troubles with MQTT.

I never used MQTT, its my first time.

I install Home Assistant using Hyper V, and install mosquitto broker inside Home Assisstant with some videos on youtube.

But im having some troubles to connecting to HA. I read some topics and said that its autodiscovery, but it cant find nothing. And, when trying to add manually, it doesnt find anything with name "layzspa"(as i read in some topics to try).

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/32704673/8ae82e1d-2696-491a-b162-bfe19c32fbc5)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/32704673/91b33d1b-5310-4aa9-a854-4f15eed6b2fc)

I also tried manually inside config of layzspa.local with local ipv4/external ipv4/ipva that hyper-v show(and that i connect to HA). I dont know what im doing wrong.

If anyone can help me, i appreciate. Thanks

**Comment by chunkysteveo** [comment]
What's the MQTT status on the Layzspa web gui? And what are your MQTT credentials/settings on the LZSpa MQTT setting page?

**Reply by ronnaudao** [reply]
I solved it just now. It has to do something with Hyper V of windows. It appears that it redirects to a fictitious IP that is only accessible from that device(177.x.x.x when local ip its 10.x.x.x). I solved installing an older version of oracle vm(since the 7.0.16 is having some troubles when configured to bridged) and it works just fine as should be.
